compiler-rt/lib/sanitizer_common/sanitizer_platform_limits_posix.h

```diff
@@ -609,7 +627,15 @@ struct __sanitizer_sigset_t {
   // The size is determined by looking at sizeof of real sigset_t on linux.
   uptr val[128 / sizeof(uptr)];
 };
-#endif
+#  elif SANITIZER_AIX
```

Comment:
Whitespace is inconsistent with line 625: https://github.com/llvm/llvm-project/blob/5e4dc56ae09f85a8ff813bfe66126921d8136ff8/compiler-rt/lib/sanitizer_common/sanitizer_platform_limits_posix.h#L625